dbms/src/Storages/DeltaMerge/RowKeyRange.h

```diff
@@ -154,13 +105,29 @@ struct RowKeyValue
         int_value = rowkey_value.int_value;
     }
 
+    // Generate from int handle
     static RowKeyValue fromHandle(Handle value)
```

Comment:
Seems this function is only used in test, just wrap it with `DBMS_PUBLIC_GTEST`?

Reply:
done